pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
```

Comment:
> You are mutating resources across the entire cluster.  I would prefer
> you leave the name as is.

So does every other admission plugin we have (`NamespaceLifecycle`, `OriginPodNodeEnvironment`, `LimitRanger`, `ServiceAccount`, ***).  None of them are selectively choosing namespaces, so they all operate on resources across the cluster.  `Cluster` in this sense is a zero entropy phrase that adds no value to understanding.